protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/PipetteFields/WellSelectionField.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import { createPortal } from 'react-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import {
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  InputField,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+import { COLUMN } from '@opentrons/shared-data'
+import {
+  actions as stepsActions,
+  getSelectedStepId,
+  getWellSelectionLabwareKey,
+} from '../../../../../ui/steps'
+import { selectors as stepFormSelectors } from '../../../../../step-forms'
+import { SelectWellsModal } from '../../../../../organisms'
+import { getMainPagePortalEl } from '../../../../../components/portals/MainPageModalPortal'
+import { getNozzleType } from '../utils'
+import type { FieldProps } from '../types'
+
+export type WellSelectionFieldProps = FieldProps & {
+  nozzles: string | null
+  pipetteId?: string | null
+  labwareId?: string | null
+}
+
+export const WellSelectionField = (
+  props: WellSelectionFieldProps
+): JSX.Element => {
+  const { t, i18n } = useTranslation('form')
+  const {
+    nozzles,
+    labwareId,
+    pipetteId,
+    onFieldFocus,
+    value: selectedWells,
+    updateValue,
+    onFieldBlur,
+    name,
+    disabled,
+    errorToShow,
+  } = props
+  const dispatch = useDispatch()
+  const stepId = useSelector(getSelectedStepId)
+  const pipetteEntities = useSelector(stepFormSelectors.getPipetteEntities)
+  const wellSelectionLabwareKey = useSelector(getWellSelectionLabwareKey)
+  const primaryWellCount =
+    Array.isArray(selectedWells) && selectedWells.length > 0
+      ? selectedWells.length.toString()
+      : undefined
```

Comment:
for consistency `protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/PipetteFields/TipWellSelectionField.tsx`

```suggestion
      : null
```